python/lsst/ts/m2/utility.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of ts_m2.
```

Comment:
Done in the commit: baf6155. I used the name of `enum.py` although I do not think this is a good idea. The python has the `enum` module already. I do not use the plural form because there is only one `enum` type now... Thanks!

Reply:
Done in the commit: baf6155. I used the name of `enum.py` although I do not think this is a good idea. The `python` has the `enum` module already. I do not use the plural form because there is only one `enum` right now... Thanks!